src/KafkaFlow.Serializer.ApacheAvro/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,20 @@
+namespace KafkaFlow.Serializer.ApacheAvro
+{
+    using System;
+    using Configuration;
+    using Confluent.SchemaRegistry;
+    using Confluent.SchemaRegistry.Serdes;
+    
+    public static class ClusterConfigurationBuilderExtensions
+    {
+        public static IClusterConfigurationBuilder WithSchemaRegistry(
```

Comment:
missing summary description, in public methods it is a must have 😃

Reply:
done